packages/dart/README.md

```diff
@@ -28,26 +30,34 @@ This library gives you access to the powerful Parse Server backend from your Dar
 
 ## Compatibility
 
-The Parse Dart SDK is continuously tested with the most recent release of the Dart framework to ensure compatibility. To give developers time to upgrade their app to the newest Dart framework, previous Dart framework releases are supported for at least 1 year after their [release date](https://dart.dev/get-dart/archive).
+The Parse Dart SDK is continuously tested with the most recent release of the Dart framework to
+ensure compatibility. To give developers time to upgrade their app to the newest Dart framework,
+previous Dart framework releases are supported for at least 1 year after
+their [release date](https://dart.dev/get-dart/archive).
 
-| Version   | Latest Version | End of Support | Compatible                                   |
-|-----------|----------------|----------------|----------------------------------------------|
-| Dart 3.0  | 3.0.0          | May 2024       | ✅ Yes                                        |
-| Dart 2.19 | 2.19.6         | Mar 2024       | ✅ Yes                                        |
-| Dart 2.18 | 2.18.7         | Jan 2024       | ✅ Yes                                        |
-| Dart 2.17 | 2.17.7         | Aug 2023       | ❌ No (Parse Dart SDK requires Dart >=2.18.0) |
+| Version   | Latest Version | End of Support | Compatible |
+|-----------|----------------|----------------|------------|
+| Dart 3.1  | 3.1.2          | Sep 2024       | ✅ Yes      |
+| Dart 3.0  | 3.0.7          | May 2024       | ✅ Yes      |
+| Dart 2.19 | 2.19.6         | Mar 2024       | ✅ Yes      |
+| Dart 2.18 | 2.18.5         | Nov 2023       | ❌ No       |
+
+> Parse Dart SDK requires **Dart >=2.19.6**
 
 ## Getting Started
 
-To install, add the Parse Dart SDK as a [dependency](https://pub.dev/packages/parse_server_sdk/install) in your `pubspec.yaml` file.
+To install, add the Parse Dart SDK as
+a [dependency](https://pub.dev/packages/parse_server_sdk/install) in your `pubspec.yaml` file.
 
 ## Documentation
 
 Find the full documentation in the [Parse Dart SDK guide][guide].
 
 ## Contributing
 
-We want to make contributing to this project as easy and transparent as possible. Please refer to the [Contribution Guidelines](https://github.com/parse-community/Parse-SDK-Flutter/blob/master/CONTRIBUTING.md).
+We want to make contributing to this project as easy and transparent as possible. Please refer to
+the [Contribution Guidelines](https://github.com/parse-community/Parse-SDK-Flutter/blob/master/CONTRIBUTING.md).

```

Comment:
There seems to be some issue with your automated code formatting (if you haven't made these changes manually). There should not be any line breaks, these are added automatically as needed by the markdown renderer.

Reply:
Yes, it changed, please check again

Reply:
looks good